docs/reference/validation.txt

```diff
@@ -23,3 +23,32 @@ Mongoid behaves slightly differently to Active Record when using ``#valid?``
 on already persisted data. Active Record's ``#valid?`` will run all
 validations whereas Mongoid's ``#valid?`` will only run validations on
 documents that are in memory as an optimization.
+
+
+``validates_uniqueness_of`` and ``:conditions``  Option
+=======================================================
+
+The ``:conditions`` option to ``validates_uniqueness_of`` can be used to
+provide additional conditions to add to the database query looking for
+identical documents. This option does not influence when the validation
+is executed because it is not considered when Mongoid retrieves the present
+value of the respective field from the model. Consider the following example:
```

Comment:
I've read this paragraph several times and I'm really struggling to understand it's meaning. I'd propose something like the following:

```
``validates_uniqueness_of`` takes a ``:conditions`` which adds a filter condition to the
query used to find existing documents in the database. This can be useful when < insert use case here >
```